Source/Resources/Content/MSGraph/DeviceAppManagement/TargetedManagedAppConfigurations/Configure Policy Managed Client Apps on Unmanaged iOS Devices.json

```diff
@@ -297,5 +297,5 @@
       "value": "true"
     }
   ],
-  "description": "Configures default behavior for application settings of policy managed client apps on unmanaged iOS devices."
+  "description": "What is it: Configures default the behavior for application settings of managed apps on unmanaged iOS devices. \n Simeon Notes: Improve your users' iOS mobile experience by expanding the list of applications that are protected by Intune and data loss protection policies. See the Intune portal for the full list of apps included in Simeon's baseline. \n Learn More: https://docs.microsoft.com/en-us/mem/intune/apps/app-configuration-policies-use-ios"
```

Comment:
**What does this do?**
Configures the default behavior for application settings of managed apps on unmanaged iOS devices. The baseline expands the list of applications that are protected by Intune and data loss protection policies. See the Intune portal for the full list of apps included in the baseline.

**Why should you use this?**
If you want to improve your users' iOS mobile experience by expanding the list of protected applications. 

**What is the end-user impact?**
 iOS device users will have more protected apps that they may use. @metamenos is this correct? that is the app is protected they may use it?

**Learn more**
[Add app configuration policies for managed iOS/iPadOS devices](https://docs.microsoft.com/en-us/mem/intune/apps/app-configuration-policies-use-ios) 

@metamenos  and is this the right page to link to? we say that this configures behavior for apps on UNMANAGED devices but the link discusses configuring policies for MANAGED devices

Reply:
For end user impact, yes it does expand the list of apps the users can use and have the organization data protected.

Good call on the link, a better one would be: https://docs.microsoft.com/en-us/mem/intune/apps/app-configuration-policies-managed-app

Reply:
@metamenos Jeff says its only about organization data, not access, because it doesn't affect whether someone can sign in to zoom app using azure ad it only affects whether or not data can leave outlook and go to the zoom app. Is this right?